### DISCUSSION TITLE
V3 und Homematic Wired Geräte

### DISCUSSION THREAD

**Discussion by Frank-2021** [Zefau/ioBroker.jarvis#1528] [General]
Hallo,

ich habe 3 x HMW-LC-Sw2-DR / HMW-Sen-SC-12-FM / HMW-IO-4-FM verbaut. Leider lassen sich diese Geräte nicht mehr Schalten.

Ist das bekannt?

Danke

Frank

**Comment by Zefau** [comment]
Ist nicht bekannt. Welche genaue Version hast du im Einsatz? Was konkret ist das Problem? Also wie drückt sich das aus, dass es sich nicht schalten lässt?
